test-common/src/main/kotlin/io/opentelemetry/android/test/common/AssertionExtensions.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.test.common
+
+import io.opentelemetry.sdk.logs.data.internal.ExtendedLogRecordData
+import io.opentelemetry.sdk.testing.assertj.LogRecordDataAssert
+import io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.assertThat
+
+fun LogRecordDataAssert.hasEventName(eventName: String): LogRecordDataAssert {
+    isNotNull()
```

Comment:
i think this is redundant since `LogRecordDataAssert.hasEventName` so `LogRecordDataAssert` is never null

Reply:
+1

Reply:
Are we sure about this? The extension is on the `LogRecordDataAssert` (the assertion class, NOT the log record itself), and the method `isNotNull()` is on the assetj parent class `AbstractAssert` and it verifies that the actual result isn't null. I expect that the error message would be easier to read by checking explicitly here, so like `assertThat(null).hasEventName("foo")` would give a clearer message 

Are we talking about the same thing here?

The `isInstanceOf()` also contains an internal `isNotNull()` check, but when the eventName comes out of incubating, then the instance checks go away and we will lose the specificity.